### PR TITLE
Stat tracker refactor

### PR DESCRIPTION
**Pull request by dunlapww**
A minor refactor consolidating 3 methods into one.  All tests still pass.

Still trying to understand the stat_tracker, but it looks like we have a class method that then calls an instance method that reads in the files.  Wondering whether we can do this all in a class method so that it's not repulling the data with every instantiation.

### CODE REVIEW

File: test/stat_tracker_test.rb [hunk 1]
```diff
@@ -3,14 +3,45 @@
...
+    expected = [:game_id, :team_id, :hoa, :result, :settled_in, :head_coach,
+      :goals, :shots, :tackles, :pim, :powerplayopportunities, :powerplaygoals,
+      :faceoffwinpercentage, :giveaways, :takeaways]
+    assert_equal expected, @stats.game_teams.headers
   end
+
+
+
+
+
```
Comment (hopesgit) [inline test/stat_tracker_test.rb:46]: There's a lot of empty space here.

**Review by hopesgit — COMMENTED** [review]
I like the new names for the files! "Test" was getting really confusing.

**Comment by jakeheft** [comment]
Great job consolidating here Will! Do we want to merge this now?

**Comment by jdcarey128** [comment]
Can we close this PR given Hope's refactor?

**Comment by hopesgit** [comment]
It could be reworked instead, but I feel that the other PR he made would be better to rework. I'm on the side of closing it.

**Pull request closed without merging**